server/etcdserver/api/v3compactor/periodic_test.go

```diff
@@ -189,7 +190,7 @@ func TestPeriodicSkipRevNotChange(t *testing.T) {
 	initialIntervals, intervalsPerPeriod := tb.getRetentions(), 10
 
 	// first compaction happens til 5 minutes elapsed
-	for i := 0; i < initialIntervals; i++ {
+	for i := 0; i < initialIntervals-1; i++ {
```

Comment:
initialIntervals is 11 while 10 is enough to trigger initial compact. In original test, the last rg.Wait(1) in initial loop always timeout. I assume this is not needed so I removed it. Otherwise the test will stuck in the last rg.Wait(1) in initial loop with this fix.